app/midle/page.tsx

```diff
@@ -1,8 +1,22 @@
+import { DocReact, allDocReacts } from '@/.contentlayer/generated';
```

Comment:
Cannot we rename `allDocReacts` to `allReactDocs`?

Reply:
Contentlayer config has got 'DocReact' name and generates automatically allDocReacts. I can guess that I should have called ReactDoc and so on. There are 
![image](https://github.com/ViktoriaAfitova/from-Zero-to-Hero/assets/90246559/a170f2f5-ed90-4755-a443-204b9166734d)

Reply:
Got it, let's leave it as it is for consistency reasons.